recipes/gerrychain/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "gerrychain" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2d3dc1ff35c6f6c1670918f30e8be4c00faed1178b19501e38bfef09c2f023b5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - geopandas
+    - matplotlib
```

Comment:
Are these packages really necessary at install time?  If so, it might be good to look at your package's structure to try to make your imports not happen when calling setup.py.

Reply:
Thanks for pointing this out---I don't think they are necessary.